### DISCUSSION TITLE
Gramps-web through Github?

### DISCUSSION THREAD

**Discussion by parthpatel1223** [gramps-project/gramps-web#319] [Q&A]
Hi, I'm an amateur to all of the technical stuff. I just want to be able to work on the same file from places other than my laptop. Is there a way to deploy an instance of it through GitHub? For example, I'd want to be able to go to a website that is spun up by my GitHub repository and do all of my work there, and have the changes I made saved, so that when I access the website from another device, my changes are there.

**Comment by DavidMStraub** [comment]
Hi, you can't run Gramps Web through Github. See here for documentation: https://www.grampsweb.org/admin/
